examples/demo_netdev_network_trunk.pp

```diff
@@ -0,0 +1,25 @@
+# Manifest to demo cisco_interface provider
```

Comment:
Would it be possible to create a `demo_all_netdev.pp` file similar to `demo_all.pp` and move all of the netdev providers including this one into the `demo_all_netdev.pp`?